assets/app/views/_templateopt.html

```diff
@@ -1,4 +1,4 @@
-<div class="template-options" ng-show="hasParameters">
+<div class="template-options" ng-show="hasParameters" ng-form="paramForm">
```

Comment:
I don't think it's a problem from my reading of the documentation for `ngForm`:

https://docs.angularjs.org/api/ng/directive/form